pwdtk/apps.py

```diff
@@ -22,49 +22,11 @@ def ready(self):
 
         """
         logger.debug("PWDTK READY")
-        if not settings.PWDTK_ENABLED:
+        if not PwdtkSettings.PWDTK_ENABLED:
             logger.debug("PWDTK DISABLED")
             return
-        if django.VERSION < (1, 11):
-            # Connect Signals
-            import pwdtk.sig_receivers  # noqa: F401
 
-            logger.debug("old django style: will decorate some functions")
+        User = get_user_model()
+        from pwdtk.watchers import watch_set_password
+        User.set_password = watch_set_password(User.set_password)
```

Comment:
decorating `set_password` is really ugly (it is a very dirty hack) and I think this should not be necessary for django >= 1.11

You found no other way for newer django versions?